packages/contracts/contracts/factory/DAOFactory.sol

```diff
@@ -8,6 +8,8 @@ import {PermissionLib} from "../core/permission/PermissionLib.sol";
 import {createERC1967Proxy} from "../utils/Proxy.sol";
 import {PluginRepo} from "../plugin/PluginRepo.sol";
 import {PluginSetupProcessor} from "../plugin/PluginSetupProcessor.sol";
+import {hHash, PluginSetupRef} from "../plugin/psp/utils/Common.sol";
```

Comment:
helperHash. It's PSP common functions. I don't like calling them `helpersHash`.

Reply:
Auditors will spend more time going back and forth to learn what this does, and little things add pretty quickly. This time would be better spent finding potential flaws instead of here. 

How about `hashHelpers(...)` or `hashedHelpers()`? 
Similar for the other abbreviated functions

Reply:
https://github.com/aragon/core/pull/164/commits/a493497c9613e0c51489048ae69ac6921171b3b9